LeaderboardBackend.Test/consts.cs

```diff
@@ -0,0 +1,7 @@
+namespace LeaderboardBackend.Test;
+
+public static class Routes
+{
+    public const string LOGIN = "/login";
+    public const string REGISTER = "/account/register";
```

Comment:
The use of `const` should be reserved for values that can never change, such as mathematical constants.

Reply:
Don't see why we can only use `const` that way. It seems more like a philosophical decision. Also, removing it will prevent us from e.g. using them as attribute arguments (it's a small example, but it's something to keep in mind)

Reply:
From Microsoft's C# guide:

> Don’t create a constant to represent information that you expect to change at any time. For example, don’t use a constant field to store the price of a service, a product version number, or the brand name of a company. These values can change over time, and because compilers propagate constants, other code compiled with your libraries will have to be recompiled to see the changes. See also the [readonly](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/readonly) keyword.

Endpoints can potentially change in the future, therefore it is not suitable to use `const` for them.

Reply:
The More You Know 🌈 🌟

Thanks for letting me know. Changing them to be `static readonly`.

Reply:
While the advice is generally correct, this a test project so the problem it avoids does not apply in this context.

Reply:
You can make the const `internal` to prevent the issue entirely.

Reply:
@TheTedder I'll revert the members to consts. And I'll also make the class itself internal, instead of the members themselves, since that makes more sense IMO.